lib/orangetheses/indexer.rb

```diff
@@ -91,6 +91,14 @@ def title(dc_elements)
       titles.empty? ? nil : titles.first.text
     end
 
+    def title_sort(dc_elements)
+      titles = dc_elements.select { |e| e.name == 'title' }
+      title = titles.empty? ? nil : titles.first.text
+      unless title.nil?
+        title.downcase.gsub(/\p{^Alnum}/, '').gsub(/^(?:a)|(?:an)|(?:the)/, '')
```

Comment:
Right, but given something like this as the title:

> "The Foo Bar", an Analysis

It's a lot easier to find the article if I normalize out the punctuation first.